code/modules/surgery/helpers.dm

```diff
@@ -80,13 +79,13 @@
 		for(var/obj/item/clothing/I in list(C.back, C.wear_mask, C.head))
 			covered_locations |= I.body_parts_covered
 			face_covered |= I.flags_inv
-			eyesmouth_covered |= I.flags_cover
+			eyesmouth_covered |= I.flags
 		if(ishuman(C))
 			var/mob/living/carbon/human/H = C
 			for(var/obj/item/I in list(H.wear_suit, H.w_uniform, H.shoes, H.belt, H.gloves, H.glasses, H.ears))
 				covered_locations |= I.body_parts_covered
 				face_covered |= I.flags_inv
-				eyesmouth_covered |= I.flags_cover
+				eyesmouth_covered |= I.flags

```

Comment:
MASKCOVERSMOUTH and other cover bitflags that we are going to check for are in flags_cover, not flags.